### PR TITLE
bumped dependency version: aiokafka==0.7.1

### PR DESCRIPTION
**Pull request by pwilczynskiclearcode**
## Description

https://github.com/aio-libs/aiokafka/blob/master/CHANGES.rst#071-2021-06-04
https://pypi.org/project/aiokafka/0.7.1/#files

This version fixes https://github.com/aio-libs/aiokafka/issues/727
```
Make sure generation and member id are correct after (re)joining group. (issue #727 and pr #747 by @vangheem)
```

### CODE REVIEW

**Comment by marcelotrevisani** [comment]
Is it possible to release another version after this fix please?

File: requirements/requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 aiohttp>=3.5.2,<4.0
 aiohttp_cors>=0.7,<2.0
-aiokafka==0.7.0
```
Comment (marcelotrevisani) [inline requirements/requirements.txt:3]: wouldn't be better to put it as 
```
aiokafka>=0.7.1,<0.8.0
```
?

Comment (patkivikram) [reply requirements/requirements.txt:3]: Could you update the PR with the requested changes? I can approve it once the updated version is pushed and do another release

**Comment by taybin** [comment]
This looks good and it passes the tests.  Merging.

**Pull request merged**